fe/fe-core/src/main/java/com/starrocks/analysis/SlotDescriptor.java

```diff
@@ -82,6 +82,14 @@ public SlotDescriptor(SlotId id, TupleDescriptor parent) {
         this.isMultiRef = false;
     }
 
+    public SlotDescriptor(SlotId id, String name, Type type, boolean isNullable) {
+        this.id = id;
+        this.label_ = name;
```

Comment:
label_ -> label

Reply:
The old codes are all label_, This PR won't change it, we could change it in another PR